pkg/terminal/command.go

```diff
@@ -218,6 +219,29 @@ Supported commands: print, stack and goroutine)`},
 	condition <breakpoint name or id> <boolean expression>.
 	
 Specifies that the breakpoint or tracepoint should break only if the boolean expression is true.`},
+		{aliases: []string{"config"}, cmdFn: configureCmd, helpMsg: `Changes configuration parameters.
+		
+	config -list
+	
+Show all configuration parameters.
+
+	config -save
+
+Saves the configuration file to disk, overwriting the current configuration file.
+
+	config <parameter> <value>
+	
+Changes the value of a configuration parameter.
+
+	config subistitute-path <from> <to>
+	config subistitute-path <from>
+	
+Adds or removes a path subistitution rule.
+
+	config alias <alias> <command>
```

Comment:
nit: I think this UX is kind of odd, mind switching it to `<command> <alias>`? Seems more intuitive to me (as in "alias 'foo' to 'bar'").